bindings/python/crocoddyl/multibody/contacts/contact-6d-loop.cpp

```diff
@@ -0,0 +1,245 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2019-2023, LAAS-CNRS, University of Edinburgh
```

Comment:
we are in 2024. The same issue is in other files.

Reply:
I fixed Copyrights where needed